src/index.html

```diff
@@ -1,20 +1,24 @@
 <!DOCTYPE html>
 <html lang="en-nz">
+
 <head>
 	<meta charset="UTF-8">
 	<title>How dead inside should you be as a Vic student?</title>
-
+	<link rel="icon" type="image/x-icon" href="icon.png">
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
+
 <body>
 	<header>
 		<h1>How dead inside should you be as a Vic student?</h1>
 	</header>
 
 	<section class="deadness">
-		It's <h2><em>Week <span class="deadness__week-number"></span></em></h2>, so you should be <span class="deadness__feeling"></span>
+		It's <h2><em>Week <span class="deadness__week-number"></span></em></h2>, so you should be <span
+			class="deadness__feeling"></span>
```

Comment:
Can you please fix this formatting so it's on one-line for single-attr inline tags?